BookingsApi/BookingsApi.DAL/Queries/GetBookingsByCaseTypesQuery.cs

```diff
@@ -146,6 +146,40 @@ private static IQueryable<VideoHearing> ApplyOptionalFilters(GetBookingsByCaseTy
 
             return hearings;
         }
+        
+        private IQueryable<VideoHearing> FilterByCaseNumber(
```

Comment:
Can you double check the performance on this query please? One of reasons we had performance issues with the booking list search was that we queried everything in memory instead of building a SQL query that could handle the filter.
Would the following below work?
```suggestion
        hearings.Where(x => x.HearingCases.Any(hc => hc.Case.Number.Contains(query.CaseNumber)));
```

Reply:
yes, good shout. I thought from the original filter that it was more complex than that, but actually, a where contains is all that's needed :)

average response time for the case number search ~ 962ms on dev dataset